bin/snarkos.rs

```diff
@@ -55,34 +55,6 @@ use snarkvm_utilities::{to_bytes_le, FromBytes, ToBytes};
 use std::{fs, net::SocketAddr, str::FromStr, sync::Arc, time::Duration};
 
 use tokio::runtime;
-use tracing_subscriber::EnvFilter;
-
-fn initialize_logger(config: &Config) {
```

Comment:
it would be great if we could move even more common, "`main`-specific" functions like this to some module containing functions usable by any node type